anitya/static/package.json

```diff
@@ -6,10 +6,6 @@
   "directories": {
     "doc": "docs"
   },
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "build": "webpack"
```

Comment:
Cleaned this up additionally. There isn't a webpack build or unit tests, so we can safely remove these lines for now.

Reply:
Nice, I forgot that we don't have this anymore